devicetypes/erocm123/inovelli-2-channel-smart-plug-mcd.src/inovelli-2-channel-smart-plug-mcd.groovy

```diff
@@ -272,10 +272,7 @@ def updated() {
 		createChildDevices()
 	} else if (device.label != state.oldLabel) {
 		childDevices.each {
-			if (it.label == "${state.oldLabel} (CH${channelNumber(it.deviceNetworkId)})") {
```

Comment:
I don't think we want to remove this check. The reason it's there is to update the label if we set it but leave it alone if the user has changed it.